TypeCobol/Compiler/CupParser/NodeBuilder/ProgramClassBuilder.cs

```diff
@@ -5,6 +5,7 @@
 using TypeCobol.Compiler.CodeModel;
 using TypeCobol.Compiler.Nodes;
 using TypeCobol.Compiler.Parser;
+using Analytics;
```

Comment:
Not required ? (I haven't checked but according to the modifications it should not)